### PR TITLE
Kysymysryhma excel

### PR DESCRIPTION
**Pull request by heeboA**
Format kysymyryhmä answers for excel export.

### CODE REVIEW

File: src/clj/ataru/applications/excel_export.clj [hunk 1]
```diff
@@ -131,18 +131,38 @@
             (str filename " (" (util/size-bytes->str size) ")")))
         value))))
 
+(defn- sec-or-vec? [value]
```
Comment (starvinraven) [inline src/clj/ataru/applications/excel_export.clj]: I think `sequential?` would be equivalent?

File: src/cljs/ataru/virkailija/editor/components/toolbar.cljs [hunk 2]
```diff
@@ -14,8 +14,7 @@
      ["Tekstikenttä" component/text-field]
      ["Tekstialue" component/text-area]
      ["Vierekkäiset tekstikentät" component/adjacent-fieldset]
-     ;["Kysymysryhmä" component/question-group] Will be enabled when feature is ready
-     ]
+     ["Kysymysryhmä" component/question-group]]
```
Comment (starvinraven) [inline src/cljs/ataru/virkailija/editor/components/toolbar.cljs]: Maybe not yet? :)

File: src/clj/ataru/applications/excel_export.clj (hunk 1 above)
Comment (heeboA) [reply src/clj/ataru/applications/excel_export.clj]: :+1:

File: src/cljs/ataru/virkailija/editor/components/toolbar.cljs (hunk 2 above)
Comment (heeboA) [reply src/cljs/ataru/virkailija/editor/components/toolbar.cljs]: Yeah, not yet..

**Pull request merged**